pkg/options/configs/dev.toml

```diff
@@ -0,0 +1,14 @@
+[services]
+solver = "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
+mediator = ["0x90F79bf6EB2c4f870365E785982E1f101E93b906"]
```

Comment:
Because I don't really know how to *use* the mediator, I can't really validate this it the right address, but we can come back to it later on when we have more context on the validator.

Reply:
Sounds good. FWIW, these were the values we had in Doppler in the `dev` environment.